### DISCUSSION TITLE
How do I check if a text column is empty?

### DISCUSSION THREAD

**Discussion by younes-alouani** [prisma/prisma#14963] [Q&A]
For a model : 

```
model Profile {
  id         String             @id @default(uuid())
  role       String?
}
```
How can I filter profiles with null/empty role ? 

`SELECT * FROM myTable WHERE myColumn IS NULL OR myColumn = ''`

**Comment by nurul3101** [comment]
Hey @younes-alouani 👋 

Here's how you can get the records which have null role.

```ts
  const profiles = await prisma.profile.findMany({
    where: {
      role: null,
    },
  });
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
